lib/msal-common/src/auth/ScopeSet.ts

```diff
@@ -147,13 +147,21 @@ export class ScopeSet {
 
     /**
      * Check if scopes intersect between this set and another.
+     * Do not allow offline_access to be the only intersecting scope
```

Comment:
I think this comment should be below on 159, along with any other nuances you think someone should know.